plugins/producer_api_plugin/producer.swagger.yaml

```
@@ -44,6 +44,39 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /producer/pause_at_block:
+    post:
+      summary: pause_at_block
+      description: |
+        Pause node at the specified block. Use /producer/resume to un-pause. If specified block is less than head then returns an error.
+        
+        ### Usage
+        
+        Note that this pauses all nodes not just producer nodes. The common use case is to use on non-producer nodes as a way of freezing state. For example, a user might wish to download all the rolls of a table. This would allow the user to freeze at a block number and then iterate through an entire table.
```

I think it should be `rows`.
```suggestion
        Note that this pauses all nodes not just producer nodes. The common use case is to use on non-producer nodes as a way of freezing state. For example, a user might wish to download all the rows of a table. This would allow the user to freeze at a block number and then iterate through an entire table.
```

---

ha!